README.md

```diff
@@ -393,6 +393,95 @@ load("@maven//:compat.bzl", "compat_repositories")
 compat_repositories()
 ```
 
+### Dependency tree analysis
+
+Each `maven_install` repository exposes a `dependency_tree.bzl` file
+containing the source of truth used by `rules_jvm_external` to generate the
+BUILD file. This source of truth comes in the form of a Starlark dictionary
+named `dependency_tree` in `@maven//:dependency_tree.bzl`.
+
+To use it, load it in your `BUILD`, `WORKSPACE` or any `.bzl` file:
+
+```python
+load("@maven//:dependency_tree.bzl", "dependency_tree")
+
+all_artifact_coordinates = [artifact["coord"] for artifact in dependency_tree]
+```
+
+The dictionary has the following schema:
+
+```python
+[
+  {
+    "coord": # Coordinates for this artifact.
+    "file": # Relative path to the artifact file.
+    "dependencies": [
+        # A list of coordinates for the dependencies of this artifact.
+    ]
+  },
+  # Other artifacts..
+]
+```
+
+To get the absolute path to the artifact, prefix the `file` value with the value of `bazel info $(output_base)/external/maven`.
```

Comment:
Sorry I know it's WIP, just a reminder here.
I suppose it should be `$(bazel info output_base)/external/maven`

Reply:
Oo, good catch.